### PR TITLE
Wizard recipe: neofetch-v6.1.0

### PR DESCRIPTION
**Pull request by innerlee**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: neofetch
* Version: v6.1.0

@staticfloat please review and merge.

### CODE REVIEW

**Comment by staticfloat** [comment]
Hmmm, this is another one of those cases where we really kind of want an `AnyPlatform`, huh.

@innerlee may I ask what it is you want to do with this?  It's just `bash` scripts, right?

**Comment by giordano** [comment]
This is probably something better suited for a simple Artifact, as there isn't much to load

**Comment by innerlee** [comment]
right, it is just a bash script.May I ask what is the AnyPlatform? Is it wider than supported_platforms?

**Comment by giordano** [comment]
> May I ask what is the AnyPlatform?

https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/455

**Comment by innerlee** [comment]
Any more info needed to provide? Do you accept this kind of `simple bash scripts`? Its okay if not, because it is not needed by any julia package.

To add some motivation for my series of prs. I use this BB system not inside julia. What I need is just the built binaries at the `*.jll` packages. I can extract them into a folder and use the programs without root permission. A very simple&naive idea.

**Comment by staticfloat** [comment]
I think once we create an `AnyPlatform` I'd be okay with having this.  It's just very wasteful for us to generate 17 tarballs for the same thing.

**Comment by innerlee** [comment]
Cool! good to know

**Comment by innerlee** [comment]
@giordano Could you help change it to `AnyPlatform`?

File: N/neofetch/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+script = raw"""
+cd $WORKSPACE/srcdir/neofetch-*/
+export PREFIX=$prefix
+make install
+install_license LICENSE.md
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```
Comment (giordano) [inline N/neofetch/build_tarballs.jl:24]:
```suggestion
platforms = [AnyPlatform()]
```

**Pull request merged**